src/cirrus/release.py

```diff
@@ -105,6 +101,52 @@ def release_branch_name(config):
     return branch_name
 
 
+def release_config(config, opts):
+    """
+    _release_config_
+
+    Extract and validate the release config parameters
+    from the cirrus config for the package
+    """
+    release_config = {
+        'wait_on_ci': False,
+        'wait_on_ci_timeout': 600,
+        'wait_on_ci_interval': 2,
+        'github_context_string': None,
+        'update_github_context': False,
+    }
+
+    if 'release' in config:
+        release_config['wait_on_ci'] = config.get_param(
+            'release', 'wait_on_ci', False
+        )
+        release_config['wait_on_ci_timeout'] = int(config.get_param(
+            'release', 'wait_on_ci_timeout', 600)
+        )
+        release_config['wait_on_ci_interval'] = int(config.get_param(
+            'release', 'wait_on_ci_interval', 2)
+        )
+        release_config['update_github_context'] = bool(config.get_param(
+            'release', 'update_github_context', False
+        ))
+        release_config['github_context_string'] = config.get_param(
+            'release', 'github_context_string', None
+        )
```

Comment:
I think you can eliminate some of these duplicated literals. Make a `release_config_defaults` dict above as you have `release_config` now. Then if "release" is in the config, iterate over the keys in `release_config_defaults` and try to get the parameter for each one, defaulting to the value in `release_config_defaults`. Add an else setting `release_config` to `release_config_defaults` if there is no section.